runbooks/how-to-create-server-regression-dashboards.md

```diff
@@ -0,0 +1,343 @@
+    11. Following instructions outlined above, change the suffix for "total_min_response_time" and "total_median_response_time" to " ms"
+    12. Change the chart title to "Minimum, Median, and Average Response Times (ms) over Time"
+    13. Ensure the x-axis title is "Date and Time (EST)"
+
+12. Click on "ADD" in the top-left
+13. Select "Add title"
+14. Set the sheet's title to "server-regression Statistics from TEST Environment 'master' Deployment Runs"
+15. On the top-right of the screen, click the share icon (next to the save and download icons). A dropdown menu should appear
+16. Click "Publish dashboard"
+17. Name the dashboard "bfd-test-server-regression"
+18. Click "Publish dashboard". A new page should load showing the completed Dashboard
+19. Click the share icon on the top-right of the screen. A dropdown menu should appear
+20. Click "Share dashboard". A new page should load with sharing options
+21. In the bottom right, under "Enable access for", select "Everyone in this account"
+22. Select "Discoverable in QuickSight" as well
+23. Close the Dashboard and return to the QuickSight "home" by clicking on "QuickSight" on the top-left of the screen
+24. Return to the "Analyses" view
+25. Open the "bfd-test-server-regression" analysis
+26. When loaded, click on the save icon on the top-right of the screen. A dialog pop-up should appear titled "Save a copy"
+27. In the input field, enter "bfd-prod-sbx-server-regression"
+28. Click "SAVE". The copied Analysis should load
+29. In the left pane, click the pencil icon next to the "Dataset" label. A dialog pop-up should appear with the title "Datasets in this analysis"
+30. Click on the vertical three-dots. A dropdown menu should appear
+31. Click on "Replace". A new dialog should appear named "Select replacement dataset"
+32. Select the "bfd-prod-sbx-server-regression" Dataset
+33. Click "Select"
+34. Click "Replace". All graphs in the Analysis should update appropriately with the new Dataset's field values
+35. Change the sheet's title to "server-regression Statistics from PROD-SBX Environment 'master' Deployment Runs"
+36. Repeat the steps outlined above to publish and share the Analsyis as a Dashboard named "bfd-prod-sbx-server-regression"
+37. Repeat the previous steps to duplicate the "bfd-prod-sbx-server-regression" Analysis. Name the duplicated analysis "bfd-prod-server-regression"
+38. Repeat the previous steps to replace the "bfd-prod-sbx-server-regression" Dataset with "bfd-prod-server-regression"
+39. Change the sheet's title to "server-regression Statistics from PROD Environment 'master' Deployment Runs"
+40. Repeat the steps outlined above to push and share the Analysis as a Dashboard named "bfd-prod-server-regression"
+
+Once the steps above are complete, there should be three new Dashboards available to view in QuickSight displaying performance metrics for each of the three environments (`TEST`, `PROD-SBX`, and `PROD`).
```

Comment:
A final screenshot of the final chart layout and maybe the list of dashboards might be nice too? Just to ensure the final thing looks as expected.

Reply:
Yeah, that's a good idea. I'll add a final screenshot beneath line 343 showing the finalized dashboard.